### ISSUE TITLE
message format broken

### ISSUE THREAD

**Issue by vn971** [majn/telegram-purple#389]
Message format seems to be broken sometimes, like on this picture (see the last message):
https://pointsgame.net/vn971/temp/2017.05.15_16:04:57_0ce3c81.png

Is it a known problem?
My plugin version is `v1.3.0.r113.g05b24fc-1`, built from source on 2017-05-01. OS is ArchLinux/GNU/Linux. Pidgin version is `2.12.0-1`.

**Comment by vn971** [comment]
Also, if I auto-complete the nick-name of `Bohdan Horbyeshko` (on the telegram-purple main chat), I can't properly type into the text field anymore. If I type in `12345` I get this:

```
Bohdan Horbyeshko‮‮‮: 12345
```
But actually, it turns out I cannot even properly type in the browser text field, so maybe it's some issue on my OS, or some strange UTF symbols (mixed with Arabic?)? I don't know myself, so just reporting as it is.

**Comment by bodqhrohro** [comment]
That's because of RTL symbol in the end of name. It deteriorates only notifications in official Telegram client because it displays name and chat message on separate lines, but in Pidgin RTL affects the chat too.

**Comment by majn** [comment]
@bodqhrohro So according to your messages in the dev chat, Telegram has forcibly removed RTL from your name, right?

**Comment by bodqhrohro** [comment]
@majn yes, looks like they added some purifying. Even if I insert RLO in the middle of the name, after saving it becomes a whitespace.
